ci-operator/step-registry/optional-operators/ci/operator-sdk/aws/optional-operators-ci-operator-sdk-aws-workflow.metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+	"path": "optional-operators/ci/operator-sdk/aws/optional-operators-ci-operator-sdk-aws-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"dptp"
+		]
+	}
+}
```

Comment:
could you please put an empty line at the end of every file ?

Reply:
This is auto generated